sphinx/themes/agogo/layout.html

```diff
@@ -15,7 +15,7 @@
       <div class="header">
         {%- if logo %}
           <p class="logo"><a href="{{ pathto(master_doc)|e }}">
-            <img class="logo" src="{{ pathto('_static/' + logo, 1)|e }}" alt="Logo"/>
+            <img class="logo" src="{{ pathto(logo, 1)|e }}" alt="Logo"/>
```

Comment:
I feel this is a breaking change to 3rd party themes. So it would be better to add another template variable like `logo_url` and keep the original `logo` variable as is.